providers/cncf/kubernetes/tests/unit/cncf/kubernetes/utils/test_pod_manager.py

```diff
@@ -59,30 +63,38 @@ def test_parse_log_line():
     assert line == log_message
 
 
+class DummyRetryState:
+    def __init__(self, exception=None):
+        # self.attempt_number = 1
+        self.outcome = mock.Mock() if exception is not None else None
+        if self.outcome:
+            self.outcome.exception = mock.Mock(return_value=exception)
+
+
+def test_get_retry_after_seconds_with_retry_after_header():
+    exc = ApiException(status=429)
+    exc.headers = {"Retry-After": "15"}
+    retry_state = DummyRetryState(exception=exc)
+    wait = get_retry_after_seconds(retry_state)
+    assert wait == 15
+
+
+@pytest.mark.parametrize(("attempt_number, expected_wait"), [(1, 1), (4, 8)])
```

Comment:
We upgraded to pytest 9.0.0 and now ruff is a bit more picky... (I believe...)
```suggestion
@pytest.mark.parametrize((("attempt_number", "expected_wait")), [(1, 1), (4, 8)])
```

Reply:
Ok I should rebased before the push to get the latest checks, fixed the missing " signs. Thanks!